examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/page/.content.xml

```diff
@@ -3,5 +3,5 @@
     jcr:description="Page Component for the Forms Components Library"
     jcr:primaryType="cq:Component"
     jcr:title="Page"
-    sling:resourceSuperType="core/fd/components/page/v1/page"
+    sling:resourceSuperType="core/fd/components/page/v2/page"
```

Comment:
is this change related to the bug we are addressing?

Reply:
Yes, and it breaks without this change. The main tag's contents are now inside of the body.html, which is a part of v2/page ([see here](https://github.com/adobe/aem-core-forms-components/pull/1194/files#diff-27af9ae2f33f90df3524749ed8dc9a6f3f77422c70494d31c2c07706eb2476b7)). I believe this was a miss in that PR.
Without this being updated in examples, edge rendition would not receive the json.